### PR TITLE
Use Sleeping UFS for DistributedCommandsCancelStatsTest

### PR DESCRIPTION
**Pull request by luzhang6**
### What changes are proposed in this pull request?
Added Sleeping UFS in DistributedCommandsCancelStatsTest 

### Why are the changes needed?
Use Sleeping UFS to make sure the operations on UFS would sleep and CANCEL operations would happen while the UFS operations are in progress.

### CODE REVIEW

File: tests/src/test/java/alluxio/client/cli/job/DistributedCommandsCancelStatsTest.java [hunk 1]
```diff
@@ -43,34 +44,23 @@
...
+        new UnderFileSystemFactoryRegistryRule(new SleepingUnderFileSystemFactory(
+            new SleepingUnderFileSystemOptions()
+                .setIsDirectoryMs(SLEEP_MS)
+                .setIsFileMs(SLEEP_MS)
+                .setGetStatusMs(SLEEP_MS)
+                .setListStatusMs(SLEEP_MS)
+                .setListStatusWithOptionsMs(SLEEP_MS)));
+
+  @Rule
+  public AuthenticatedUserRule mAuthenticatedUser = new AuthenticatedUserRule(TEST_USER,
```
Comment (yuzhu) [inline tests/src/test/java/alluxio/client/cli/job/DistributedCommandsCancelStatsTest.java:62]: why is this necessary?

Comment (luzhang6) [reply tests/src/test/java/alluxio/client/cli/job/DistributedCommandsCancelStatsTest.java:62]: Followed existing tests like LoadMetadataIntegrationTest and ConcurrentDeleteIntegrationTest etc, which all have AuthenticatedUserRule defined.

**Comment by yuzhu** [comment]
checkstyle @luzhang6

**Comment by yuzhu** [comment]
known test failure ```alluxio.client.fs.concurrent.ConcurrentFileInStreamIntegrationTest.FileInStreamConcurrency```

**Comment by yuzhu** [comment]
alluxio-bot, merge this please
